packages/storage/src/service.ts

```diff
@@ -160,6 +172,23 @@ export class StorageService implements _FirebaseService {
     }
   }
 
+  get emulatorOrigin(): string | undefined {
+    return this._emulatorOrigin;
+  }
+
+  /**
+   * Set emulator origin string for this service.
+   * @param origin Origin string in the form of http://[host]:[port]
+   */
+  set emulatorOrigin(origin: string | undefined) {
```

Comment:
Do we need to make this callable with `undefined`?

Reply:
Doesn't seem so, removed.